src/main/java/net/sf/jabref/gui/maintable/MainTableSelectionListener.java

```diff
@@ -407,6 +408,9 @@ private void showIconRightClickMenu(MouseEvent e, int row, MainTableColumn colum
                             }
                             menu.add(new ExternalFileMenuItem(panel.frame(), entry, content.get(), content.get(), icon,
                                     panel.getBibDatabaseContext(), field));
+                            if (field == FieldName.DOI) {
```

Comment:
Always tricked by this equals vs == in Java....when do we finally port JabRef to the nice C# world 😄